schedule/virt_autotest/default_install_svirt.yaml

```diff
@@ -0,0 +1,41 @@
+name:           default_install_svirt
+description:    >
+    Maintainer: nan.zhang@suse.com
```

Comment:
please also add our qe-virt ML to maintianer

Reply:
@nanzhg I guess you did not notice the two comments about updating maintainers? Let's quickly fix them and merge the PR :)

Reply:
I've updated maintainer in my local, but this PR still has issues during testing, I'm working on the fix before merge. :)